package.json

```diff
@@ -101,13 +101,6 @@
     "@babel/preset-typescript": "7.16.0",
     "@deanwhillier/jest-matchmedia-mock": "1.2.0",
     "@hot-loader/react-dom": "16.13.0",
-    "@storybook/addon-actions": "6.3.2",
-    "@storybook/addon-knobs": "6.3.0",
-    "@storybook/addon-links": "6.3.2",
-    "@storybook/addon-notes": "5.3.21",
-    "@storybook/addons": "6.3.2",
-    "@storybook/builder-webpack5": "6.3.2",
-    "@storybook/manager-webpack5": "6.3.2",
     "@storybook/react": "6.2.8",
```

Comment:
removing `@storybook/react` introduces new eslint failures.

Reply:
Doing following solved
- adding `buffer` fallback for webpack
- adding `@types/webpack-env`